machine.go

```diff
@@ -301,9 +301,6 @@ Type=ether
 
 [Network]
 DHCP=ipv4
```

Comment:
does this work on networks with ipv6 ?

Reply:
Yes. For IPv6, we typically do not use DHCP but ICMPv6 based autoconfiguration. We need a link local IPv6 address to perform this auto configuration and we need to accept router advertisements to configure a gateway.

Reply:
Maybe your question really was whether it also works on networks with IPv4 or with both IPv4 and IPv6. In those cases, DHCP works for IPv4 as before and the fakemachine may have dual stack network connectivity preferring IPv6.